feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -178,7 +182,7 @@ func bgpCreateNbr(localAs, peerAs uint32, policy string) *telemetry.NetworkInsta
 		}
 
 	}
-	return bgp
+	return ni_proto
```

Comment:
It wasn't evident to me why we changed the return path here? Is it because the protocol wasn't configured?

Reply:
if we don't change  old L153 to new L153+L154, i guess we don't need to change this return path as well..

Reply:
We're returning it since the protocol wasn't configured for `name` and `identifier`.